nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/impl/ReconnectedEventBundleImpl.java

```diff
@@ -82,15 +84,15 @@ public ReconnectedEventBundleImpl(EventBundle sourceEventBundle, String listener
         this.listenerName = listenerName;
     }
 
-    protected CoreSession getReconnectedCoreSession(String repoName) {
+    protected CoreSession getReconnectedCoreSession(String repoName, String originatingUsername) {
```

Comment:
We may want to keep the existing `#getReconnectedCoreSession(String)` method for backward compatibility?
WDYT @efge ?

Reply:
Yes we must have 100% compatibility in backports.